### DISCUSSION TITLE
having two field with not similar number

### DISCUSSION THREAD

**Discussion by SarvinTorkaman** [json-schema-org/community#625] [Q&A]
Hi, how can I use JSON schema validation to prevent two number input field have the same number entered

**Comment by gregsdennis** [comment]
You can't do this with JSON Schema out of the box.  You'll need extended functionality to do that, like what my [`data` vocabulary](https://docs.json-everything.net/schema/vocabs/data-2023/) provides.

Here's an [example](https://docs.json-everything.net/schema/examples/data-ref/#example-schemadata-comparative) similar to what you're trying to do.

Unfortunately, it doesn't seem my vocabulary extension has seen very wide adoption.  The only implementation I'm aware of is mine in .Net.  You can play with it at https://json-everything.net if you want.

**Reply by SarvinTorkaman** [reply]
thanks, found a way to solve this issue. used both exclusiveMaximim and exclusiveMinimum to make it uneven

**Reply by gregsdennis** [reply]
I'm glad you found a solution.

It'd be great if you shared more detail on the problem and solution for any readers who come along later!

**Reply by SarvinTorkaman** [reply]
```

      then: {
        required: ['from', 'to'],
        oneOf: [
          {
            properties: {
              to: { exclusiveMinimum: { $data: '1/from' } },
            },
          },
          {
            properties: {
              to: { exclusiveMaximum: { $data: '1/from' } },
            },
          },
```
          
          
          with this method, it will not accept the same value for from and to. but you have to change the error message somewhere else. because the error message you get is not correct
